drivers/pcie/host/shell.c

```diff
@@ -94,6 +167,38 @@ static void show_bars(const struct shell *sh, pcie_bdf_t bdf)
 	}
 }
 
+static void show_capabilities(const struct shell *sh, pcie_bdf_t bdf)
+{
+	struct pci_cap_id_to_str *cap_id2str;
+	uint32_t base;
+
+	shell_fprintf(sh, SHELL_NORMAL, "    PCI capabilities:\n");
+
+	cap_id2str = pci_cap_list;
+	while (cap_id2str->str != NULL) {
+		base = pcie_get_cap(bdf, cap_id2str->id);
+		if (base != 0) {
+			shell_fprintf(sh, SHELL_NORMAL,
+				      "        %s\n", cap_id2str->str);
+		}
+
+		cap_id2str++;
+	}
+
+	shell_fprintf(sh, SHELL_NORMAL, "    PCIe capabilities:\n");
+
+	cap_id2str = pcie_cap_list;
```

Comment:
hard to understand that this is extended and not express

Reply:
indeed, I'll change to pcie_cap_list and pcie_ext_cap_list. "extended" is about the capabilities.
Actually, I'll ditch the pci_ to pcie_ 

In fact, we should not refer to PCI anymore, PCIe is meant to be retro-compatible and we only implement PCIe specs.